packaging/windows/scripts/passwordless_share_disable.ps1

```diff
@@ -0,0 +1,50 @@
+# Following code is taken from https://mattmuster.com/2019/05/16/toggle-password-protected-sharing-on-windows-10/
+
+$script = $MyInvocation.MyCommand.Path
+If (([Security.Principal.WindowsIdentity]::GetCurrent()).Name -eq "NT AUTHORITY\SYSTEM")
+{
+    # Add the network access permission for the Everyone group
+    # This allows the radio button to be flipped
+    $everyonepath='HKLM:\SECURITY\Policy\Accounts\S-1-1-0\ActSysAc'
+    $ekv=(Get-ItemProperty $everyonepath).'(Default)'
+    $eBit=2
+
+    # Add the share permission for the Guest user
+    # This flips the radio button
+    $guestSID=(Get-Localuser | Where { $_.SID -match ".*-501" }).SID.Value
+    $guestpath="HKLM:\SECURITY\Policy\Accounts\$guestSID\ActSysAc"
+    # Guest account must be enabled to allow no password setting
+    Get-LocalUser -SID $guestSID | Enable-LocalUser
+    $gkv=(Get-ItemProperty $guestpath).'(Default)'
+    $gBit=128
+
+    # Bit flips
+    $ekv[0] = $ekv[0] - $eBit # Disallow flip
+    if (($gkv[0] -band $gBit) -eq 0) # If radio button is flipped on already
+    {
+        $gkv[0] = $gkv[0] + $gBit # Flip it off
+    }
+
+    New-ItemProperty -Path $everyonepath -Name '(Default)' -PropertyType None -Value $ekv -Force
+    New-ItemProperty -Path $guestpath -Name '(Default)' -PropertyType None -Value $gkv -Force
+}
+Else
+{
+    try
+    {
+        $taskname = "temp_ToggleNoPWShare"
+        $taskdescription = $taskname
+        $arguments = "-Executionpolicy bypass -NoProfile -file `"$script`""
+        If ($Reboot) { $arguments = $arguments + " -Reboot" }
+        $action = New-ScheduledTaskAction -Execute 'Powershell.exe' -Argument $arguments
+        Register-ScheduledTask -Action $action -TaskName $taskname -Description $taskdescription -User "System"
+        Start-ScheduledTask -TaskName "$taskname"
+        Sleep 5
+        Unregister-ScheduledTask -TaskName $taskname -Confirm:$false
+    }
+    catch
+    {
+        Write-Host "Exception $_"
+        Get-Variable -Scope Script
+    }
+}

```

Comment:
Not reviewed and tested but I am wondering, can we make this as part of `crc setup` and try to avoid put more stuff to installer side?

Reply:
we can put them in `crc setup` but these commands need admin privileges to run so to avoid the UAC prompts its added to the msi